editor/block-settings-menu/style.scss

```diff
@@ -1,15 +1,29 @@
 .editor-block-settings-menu {
 	position: absolute;
 	top: 0;
-	right: 0;
-	height: $text-editor-font-size * 4; // same height as an empty paragraph
+	right: 4px;
 	padding: 6px 0 6px 14px; // handles hover area
 
 	// Mobile
 	display: none;
 
 	@include break-small {
-		display: block;
+		display: flex;
+		flex-direction: column;
+	}
+}
+
+.editor-block-settings-menu__content {
+	margin-top: 8px;
+}
+
+.editor-block-settings-menu__toggle {
+	transform: rotate(90deg);
```

Comment:
Do we add spaces inside all parentheses also in CSS? I'm not sure, just checking. If yes, what has happened to you style linter? :)

Reply:
We don't have a style linter?

Reply:
Haha, okey :)